app/handlers/identities_forgot_password.rb

```diff
@@ -17,7 +17,8 @@ def authorized?
 
   def handle
     username = forgot_password_params.username
```

Comment:
Can you change this to username_or_email (in the form, paramify params, and this local variable)?